src/AppBundle/Resources/views/Pa/Settings/profilePasswordEdit.html.twig

```diff
@@ -0,0 +1,29 @@
+{% extends 'AppBundle:Layouts:application.html.twig' %}
+
+{% import 'AppBundle:Macros:macros.html.twig' as macros %}
+
+{% set translationDomain = "pa-profile" %}
+{% trans_default_domain translationDomain %}
+
+{% block htmlTitle %}Settings - change password{% endblock %}
```

Comment:
it'd be nice to have those into the translation file, more aligned with the rest of PA templates